frontend/__tests__/components/chat-message.test.tsx

```diff
@@ -70,4 +70,12 @@ describe("ChatMessage", () => {
     );
     expect(screen.getByTestId("custom-component")).toBeInTheDocument();
   });
+
+  it("should apply correct styles to inline code", () => {
+    render(<ChatMessage type="assistant" message="Here is some `inline code` text" />);
+    const codeElement = screen.getByText("inline code");
+    
+    expect(codeElement.tagName.toLowerCase()).toBe("code");
+    expect(codeElement.closest("article")).not.toBeNull();
+  });
```

Comment:
The test title "should apply correct styles to inline code" is unrelated to what the test actually does (check if the elements exist)

```suggestion
  it("should render inline code", () => {
    const { rerender } = render(
      <ChatMessage type="assistant" message="Here is some regular text" />,
    );

    expect(screen.queryByRole("code")).not.toBeInTheDocument();

    rerender(
      <ChatMessage
        type="assistant"
        message="Here is some `inline code` text"
      />,
    );

    expect(screen.getByRole("code")).toBeInTheDocument();
  });
```

Here we check for the presence of a `code` element only if the content contains backticks. We remove the check for `article` since that is the message wrapper which we test various times in previous tests.

If we want to test styles, I believe we can take advantage of [snapshot testing](https://vitest.dev/guide/snapshot)